content/en/docs/ambient/usage/l7-features/index.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Use Layer 7 features
+description: Supported features when using a L7 waypoint proxy.
+weight: 50
+owner: istio/wg-networking-maintainers
+test: no
+---
+
+With a waypoint proxy deployed for a workload, you can enable more of [Istio's features](/docs/concepts).
+
+Ambient mode supports configuring waypoints using the Kubernetes Gateway API. Configurations that apply to a Gateway API are called _policies_.
+
+{{< warning >}}
+The Istio traffic management APIs (virtual service, destination rules etc) are not supported by the ambient data plane mode.
```

Comment:
```suggestion
The Istio classic traffic management APIs (virtual service, destination rules etc) remains at alpha for the ambient data plane mode.
```

Reply:
Maybe just add them to the table below?

Reply:
We would then have to call out the attachment type being a selector as opposed to a targetRef.  I'd love this to be in the documentation but it's too complicated to add in this PR imo.